src/System.IO.FileSystem/tests/File/Copy.cs

```diff
@@ -56,9 +56,9 @@ public void CopyOntoSelf()
         [PlatformSpecific(TestPlatforms.AnyUnix)]
         public void DanglingSymlinkCopy()
         {
-            string dangling_symlink = GetTestFileName();
-            string missing_target = GetTestFileName();
-            string dangling_symlink_new_location = GetTestFileName();
+            string dangling_symlink = GetTestFilePath();
+            string missing_target = GetTestFilePath();
+            string dangling_symlink_new_location = GetTestFilePath();
             Assert.False(File.Exists(missing_target));
             Assert.Equal(symlink(missing_target, dangling_symlink), 0);
```

Comment:
since I just saw it, the order is wrong here:

```suggestion
             Assert.Equal(0, symlink(missing_target, dangling_symlink));
```